selfservice/strategy/oidc/provider_config.go

```diff
@@ -53,6 +53,12 @@ type Configuration struct {
 	//
 	// It can be either a URL (file://, http(s)://, base64://) or an inline JSONNet code snippet.
 	Mapper string `json:"mapper_url"`
+
+	// RequestedClaims string encoded json object that specifies claims and optionally their properties which should be
+	// included in the id_token or returned from the UserInfo Endpoint.
+	//
+	// More information: https://openid.net/specs/openid-connect-core-1_0.html#ClaimsParameter
+	RequestedClaims string `json:"requested_claims"`
```

Comment:
```suggestion
	RequestedClaims json.RawMessage `json:"requested_claims"`
```

Reply:
Yeah, that's much better. I was also thinking about how I could validate that the parameter is indeed a JSON object, but my current Go knowledge hindered me from doing so :smile:

Reply:
Done.